spec/unit/shoulda/matchers/action_controller/callback_matcher_spec.rb

```diff
@@ -45,12 +45,92 @@ def match
 
         expect { expect(controller).not_to match }.to fail_with_message(message)
       end
+
+      context "qualified with only" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.